samtranslator/translator/translator.py

```diff
@@ -332,6 +332,15 @@ def _get_embedded_connectors(self, resources: Dict[str, Any]) -> List[Resource]:
                         connector_dict,
                     )
 
+                    # In case the generated resource already contains Metadata, we need to append to it
+                    try:
+                        original_metadata = generated_connector.get_resource_attribute("Metadata")
+                    except KeyError:
+                        original_metadata = {}
+                    generated_connector.set_resource_attribute(
+                        "Metadata", {**original_metadata, "aws:sam:connectors:type": "Embedded"}
```

Comment:
Why is there both `aws:sam:connectors:type` and `ConnectorType`?

Reply:
`aws:sam:connectors:type` is used to distinguish it's a embedded connector type and later deleted.